src/App.vue

```diff
@@ -226,13 +226,12 @@ export default defineComponent({
     // get resource, ensure resource url is not empty!
     if (this.url != null && this.url != undefined && this.url != '') {
       this.dicomUrl = await this.addWadouriPrefix(this.url)
-    }
-
-    // get vip metadata
-    await this.fetchVipMetadataInformation(await this.addWadouriPrefix(this.url))
+      // get vip metadata
+      await this.fetchVipMetadataInformation(this.dicomUrl)
```

Comment:
did you confirm that these changes don't affect the functionality in the webUI?

Reply:
although I haven't built the webUI with the suggested changes on my device,  the code changes look fine to me

Reply:
yes, these changes are working fine